jcheck/src/main/java/org/openjdk/skara/jcheck/CopyrightFormatCheck.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.skara.jcheck;
+
+import org.openjdk.skara.census.Census;
+import org.openjdk.skara.vcs.Commit;
+import org.openjdk.skara.vcs.ReadOnlyRepository;
+import org.openjdk.skara.vcs.openjdk.CommitMessage;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Iterator;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class CopyrightFormatCheck extends CommitCheck {
+
+    private final ReadOnlyRepository repo;
+
+    CopyrightFormatCheck(ReadOnlyRepository repo) {
+        this.repo = repo;
+    }
+
+    @Override
+    Iterator<Issue> check(Commit commit, CommitMessage message, JCheckConfiguration conf, Census census) {
+        var metadata = CommitIssue.metadata(commit, message, conf, this);
+        var copyrightConf = conf.checks().copyright();
+        if (copyrightConf == null) {
+            return iterator();
+        }
+        var pattern = Pattern.compile(copyrightConf.files());
```

Comment:
Could use a more descriptive name.
```suggestion
        var filesPattern = Pattern.compile(copyrightConf.files());
```